File: java/bus.java
import java.util.Scanner;

public class bus {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int T = scan.nextInt();
		while (T-- > 0)
			System.out.println((int) Math.pow(2, scan.nextInt()) - 1);
		scan.close();
	}
}

File: ts/src/bus.ts
import readline = require('readline')

let lineCount = 0

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    if (lineCount > 0) {
      console.log(Math.pow(2, parseInt(line)) - 1)
    }
    lineCount++
  })
  .on('close', () => {
    process.exit(0)
  })
